File: Backend/src/main/java/de/hft_stuttgart/winf/proj2/sp/backend/dto/SaveExamAndQuestionsDTO.java
package de.hft_stuttgart.winf.proj2.sp.backend.dto;

import lombok.*;

import java.util.List;

@AllArgsConstructor
@Getter
@Setter
@ToString
@EqualsAndHashCode
public class SaveExamAndQuestionsDTO {
    private ExamDto exam;
    private List<ExamQuestionDTO> questions;

    public SaveExamAndQuestionsDTO(){}
}

File: Frontend/SPFrontend/src/app/components/models/SaveExamAndQuestionsDTO.ts
import {QuestionDto} from "./QuestionDto";
import {ExamDTO} from "./ExamDTO";

export class SaveExamAndQuestionsDTO {
  exam: ExamDTO;
  questions: QuestionDto[];

  constructor(exam: ExamDTO, questions: QuestionDto[]) {
    this.exam = exam;
    this.questions = questions;
  }

}
